### DISCUSSION TITLE
mysql vs mongo

### DISCUSSION THREAD

**Discussion by ielum** [microrealestate/microrealestate#82] [Q&A]
I see this runs off MongoDB 

why mongo over RDS like mysql? 

can microrealestate be easily modified to be backed by mysql ?

**Comment by camelaissani** [comment]
Using mongo is a bad decision made many years ago.

At this time I wanted to experiment it because it was the hype.
Also I was not sure to build something serious with this app.

A rdbms is definitely more appropriate for consistency of the data and to easily build relationship between tables.

A migration to MySQL is not impossible but something that will take time to do as it impacts all the services.
